xwiki-platform-core/xwiki-platform-rendering/xwiki-platform-rendering-macros/xwiki-platform-rendering-macro-include/src/main/java/org/xwiki/rendering/macro/include/IncludeMacroParameters.java

```diff
@@ -152,6 +160,29 @@ public String getSection()
         return this.section;
     }
 
+    /**
+     * @param isExcludeFirstHeading {@code true} to remove the first heading found inside
+     *        the document or the section, {@code false} to keep it 
+     * @since 12.4RC1 
+     */
+    @Unstable
+    @PropertyName("Exclude First Heading")
+    @PropertyDescription("Exclude the first heading from the included document or section.")
+    public void setExcludeFirstHeading(boolean isExcludeFirstHeading)
+    {
+        this.isExcludeFirstHeading = isExcludeFirstHeading;
+    }
+    
+    /**
+     * @return whether to exclude the first heading from the included document or section, or not.
+     * @since 12.4RC1 
+     */
+    @Unstable
+    public boolean isExcludeFirstHeading()
+    {
+        return this.isExcludeFirstHeading;
```

Comment:
I did not talked about the field for which starting with "is" does not make sense, like you would not start it with "get".

Reply:
I thought for boolean "is" would be better as "is True" or "is False". But `getExcludedFirstHeading` will be better to make it similar to other getters.

Reply:
Doing it

Reply:
Again is is right for the `method` but not for the `field`.

Reply:
Got it. Making changes. Thank you

Reply:
Now the function name is `isExcludeFirstHeading` and property name is `getExcludeFirstHeading`.